### ISSUE TITLE
[Enhancement]: Bookmarks in Podcasts

### ISSUE THREAD

**Issue by zblesk** [advplyr/audiobookshelf#884]
### Describe the feature/enhancement

Please add the ability to add bookmarks for podcast episodes, the same way they work for books now. 

See also: #796. 

Thank you.

**Comment by watsonbox** [comment]
I would also love to see the bookmarking feature available for Podcasts as well 👍

**Comment by revilo951** [comment]
Yes! This, so much.

**Comment by pharmacologic** [comment]
I would love this feature too. I have some good 3 hour podcast episodes I like to refer back to as an educational resource, and being able to save timestamps for the key parts within audiobookshelf would be wonderful.

**Comment by flintstones-fred** [comment]
This would be an amazing feature. I regularly listen to educational podcasts on the go and find myself wanting to listen to specific sections of the podcast later to take notes (or refer those sections to my significant other).

**Comment by zblesk** [comment]
Hey guys, in case you're interested, the Pocket Casts app just added this feature - paid tier only, though.

**Comment by Wolfbeta** [comment]
I didn't realize bookmarks for podcasts weren't implemented until I found this thread. 

Please consider increasing the priority on this enhancement. It's a must-have.

**Comment by ameyp** [comment]
Yes, please add this. Would be great to have this feature parity across audiobooks and podcasts.

**Comment by oderyn** [comment]
Similar to others in this thread, I only just discovered that podcasts don’t have bookmarking. This would be an incredibly useful feature.

**Comment by zblesk** [comment]
Not sure y'all have seen the linked issue, but advplyr said there that it probably wouldn't be too hard to implement, but he's not sure if he wants to go that way. Since this has been open for almost three years, maybe the priorities really do lie elsewhere.

**Comment by GalacticCheat** [comment]
I would like to use this feature too.
I really don't understand how this problem can last for so long, knowing that this functionality is already integrated into audiobooks and that it just needs to be integrated into podcasts as well!
